chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/ListingDetailsNormalizer.java

```diff
@@ -47,6 +48,7 @@ public ListingDetailsNormalizer(CertificationEditionNormalizer editionNormalizer
         this.editionNormalizer = editionNormalizer;
         this.acbNormalizer = acbNormalizer;
         this.atlNormalizer = atlNormalizer;
+        this.rwtNormalizer = new RwtNormalizer();
```

Comment:
Why is this the only one that's marked with `new`?

Reply:
It doesn't have dependencies on any of our other spring components (DAOs, Services, etc), so it's not a spring component (with Autowired constructor and Component annotation). Spring won't create it so I have to.